### ISSUE TITLE
Open popup for config export in addition to the download

### ISSUE THREAD

**Issue by flekschas** [higlass/higlass#33]
_Commit_: 

#### Steps to reproduce
1. Click on _export view config_

#### Observed behavior
- Downloads a JSON file

#### Expected behavior
- It would be great if one first gets a popup that shows the config with an additional option to download. When you're playing around with higlass configs often you just want to know how somebody else has configures something without having to download and open the downloaded file all the time.
- Basically something similar to what the _edit_ button provided before
